integration/mediation-tests/tests-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/clients/jmsclient/JMSQueueMessageProducer.java

```diff
@@ -0,0 +1,209 @@
+            //setting property for Qpid running on WSO2 MB
+            properties.put("connectionfactory.QueueConnectionFactory", jmsBrokerConfiguration.getProviderURL());
+        } else {
+            //setting property for ActiveMQ
+            properties.setProperty(Context.PROVIDER_URL, jmsBrokerConfiguration.getProviderURL());
+        }
+        Context context = new InitialContext(properties);
+        connectionFactory = (QueueConnectionFactory) context.lookup("QueueConnectionFactory");
+    }
+
+    /**
+     * Method to establish the connection with the given Queue, with messages not being persisted.
+     * This must be called before calling pushMessage() to send messages.
+     *
+     * @param queueName name of the Queue
+     * @throws JMSException if an error occurs creating the connection, session or producer
+     */
+    public void connect(String queueName) throws JMSException {
+        connect(queueName, false);
+    }
+
+    /**
+     * Method to establish the connection with the given Queue, with message persistance as specified.
+     * This must be called before calling pushMessage() to send messages.
+     *
+     * @param persistMessage whether or not messages need to be persisted
+     * @param queueName name of the queue
+     * @throws JMSException if connection to the queue fails
+     */
+    public void connect(String queueName, boolean persistMessage) throws JMSException {
+        connection = connectionFactory.createConnection();
+        connection.start();
+        session = connection.createSession(false, Session.AUTO_ACKNOWLEDGE);
+        Destination destination = session.createQueue(queueName);
+        producer = session.createProducer(destination);
+        if (persistMessage) {
+            producer.setDeliveryMode(DeliveryMode.PERSISTENT);
+        } else {
+            producer.setDeliveryMode(DeliveryMode.NON_PERSISTENT);
+        }
+    }
+
+    /**
+     * Method to disconnect the connection with the given Queue.
+     * This must be called after sending the messages to release the connection.
+     */
+    public void disconnect() {
+        if (producer != null) {
+            try {
+                producer.close();
+            } catch (JMSException e) {
+                //ignore
+            }
+        }
+        if (session != null) {
+            try {
+                session.close();
+            } catch (JMSException e) {
+                //ignore
+            }
+        }
+        if (connection != null) {
+            try {
+                connection.close();
+            } catch (JMSException e) {
+                //ignore
+            }
+        }
+    }
+
+    /**
+     * Method to send the message to the destination Queue.
+     *
+     * @param message message to be sent
+     */
+    public void sendMessage(Message message) throws JMSException {
+        checkIfConnected();
+        producer.send(message);
+    }
+
+    /**
+     * Method to send a TextMessage.
+     *
```

Comment:
Remove extra line

Reply:
Shouldn't there be an extra line between the method description and the parameters?